control/control_grpc_server_test.go

```diff
@@ -101,13 +101,13 @@ func TestGRPCServerScheduler(t *testing.T) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	c.Load(passthru)
+	catalogedPassthru, _ := c.Load(passthru)
 	<-lpe.done
 	filepub, err := core.NewRequestedPlugin(path.Join(fixtures.SnapPath, "plugin", "snap-plugin-publisher-mock-file"))
 	if err != nil {
 		log.Fatal(err)
 	}
-	c.Load(filepub)
+	catalogedFile, _ := c.Load(filepub)
```

Comment:
Should we grab/process the errors here just in case the `Load()` fails for some reason?